build.bat

```diff
@@ -1 +1,9 @@
+cd ./buildtab
+cmd /c "npm i && npm run build"
```

Comment:
Why not `yarn` (which is used for the resto of the setup)?

Reply:
No particular reason. I will change it to use `yarn` as well.

Reply:
Done.